src/constants/global.js

```diff
@@ -0,0 +1,6 @@
+const adminRole = 'Admin';
```

Comment:
Use capital letters for constants, like ADMIN_ROLE

Reply:
done